### PR TITLE
Fix: Log always write the same correlator id and transaction id

### PR DESCRIPTION
**Pull request by DeepshikhaNEC**
Fixed issue #326 for iotagent-ul that logs always write the same correlator id and transaction id.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -38,7 +38,7 @@
     "body-parser": "1.18.3",
     "dateformat": "3.0.3",
     "express": "~4.16.4",
-    "iotagent-node-lib": "git://github.com/telefonicaid/iotagent-node-lib.git#master",
+    "iotagent-node-lib": "git://github.com/DeepshikhaNEC/iotagent-node-lib.git#nodelib-issue/426",
```
Comment (fgalan) [inline package.json:41]: PR https://github.com/telefonicaid/iotagent-node-lib/pull/864 has been just merged into master, so please revert this change.

Comment (DeepshikhaNEC) [reply package.json]: > PR [telefonicaid/iotagent-node-lib#864](https://github.com/telefonicaid/iotagent-node-lib/pull/864) has been just merged into master, so please revert this change.

@fgalan  Hi, I have been reverted the changes of package.json
